doc/website/getting-started/overview.md

```diff
@@ -11,637 +11,220 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
 received by subscribers of the same topic. To enable publishers to offer their topic and subscribers to subscribe to
 these offered topics, the middleware daemon, called ``RouDi``, must be running.
 
-For further information how iceoryx can be used see the [examples](../../../iceoryx_examples/README.md). The
-[conceptual guide](../../conceptual-guide.md) provides additional information about the _Shared Memory
-communication_ that lies at the heart of iceoryx.
+For further information how iceoryx can be used see the 
+[examples](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/README.md). The
+[conceptual guide](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/conceptual-guide.md) provides additional 
+information about the _Shared Memory communication_ that lies at the heart of iceoryx.
 
-We now briefly define the main entities of an iceoryx system before showing how they are created and used by the
-iceoryx API.
+### icehello example
```

Comment:
An overview graphic would be cool, but I'm not sure if I can provide one until Thursday. I suggest to add the graphic within a separate PR so that we can merge this one until Thursday.

Reply:
Please add a todo with `<!-- ... -->` for the graphic